sklearn/linear_model/_ransac.py

```diff
@@ -721,3 +721,8 @@ def get_metadata_routing(self):
             .add(caller="predict", callee="predict"),
         )
         return router
+
+    def __sklearn_tags__(self):
+        tags = super().__sklearn_tags__()
+        tags.input_tags.sparse = True
```

Comment:
I agree that `fit` involved this change. However, I'm wondering if we should not instead delegate to the underlying estimator instead: if the underlying estimator does not support it, this `RANSACRegressor` will fail (e.g. `LinearRegression(positive_only=True)`).

Reply:
done in 8f2f3db